uber/config.py

```diff
@@ -412,3 +412,9 @@ def _is_intstr(s):
     c.VOLUNTEER_CHECKLIST = [url for step, url in _items]
 
 stripe.api_key = c.STRIPE_SECRET_KEY
+
+if c.KIOSK_CC_ENABLED:
+    c.DOOR_PAYMENT_METHOD_OPTS.append([c.STRIPE, "Pay with credit card now (faster)"])
+
+if c.GROUPS_ENABLED:
+    c.DOOR_PAYMENT_METHOD_OPTS.append([c.GROUP, "Taking an unassigned Group badge (group leader must be present)"])
```

Comment:
I'm guessing ``c.GROUP`` and ``c.STRIPE`` are defined in other enums and therefore it is guaranteed to be safe to reference them here?